### PR TITLE
version and time fix

### PR DESCRIPTION
**Pull request by hatsune-no-pause582**
1.12.2 to 1.17
opening hour fix

### CODE REVIEW

File: others.html [hunk 1]
```diff
@@ -60,12 +60,12 @@ <h1>回饋與建議 Feedback</h1>
...
 			<h1>多人伺服器 Server</h1>
 			<p>專屬於南科人的Minecraft Server。在這裡沒有陌生人，大家都是熟悉的南科人。稱為 <strong>NNKIEH_Land。</strong></p>
-			<p>目前屬於測試階段，不持續開機。正式開幕日期為 7/4(日)。</p>
+			<p></p>
 			<h2>資訊 Info</h2>
 			<ul>
 				<li>Server IP：nnkiehland.ddns.net</li>
-				<li>版本：1.12.2</li>
-				<li>開機時間：每日 7:00 至 22:00 (暑假暫定)</li>
+				<li>版本：1.17</li>
```
Comment (pause582) [inline others.html]: Better warning that 1.17 != 1.17.1

Comment (hatsune-no-pause582) [reply others.html]: ok

Comment (hatsune-no-pause582) [reply others.html]: will u add it to me

**Comment by pause582** [comment]
good advise

File: others.html (hunk 1 above)
Comment (pause582) [reply others.html]: ok

File: others.html [hunk 2]
```diff
@@ -64,7 +64,7 @@ <h1>多人伺服器 Server</h1>
 			<h2>資訊 Info</h2>
 			<ul>
 				<li>Server IP：nnkiehland.ddns.net</li>
-				<li>版本：1.17</li>
+				<li>版本：1.17 (注意：1.17.1與1.17不相容)</li>
```
Comment (hatsune-no-pause582) [inline others.html:67]: cool, let's merge them

Comment (pause582) [reply others.html:67]: 39 very much

**Pull request merged**